react/Overlay.tsx

```diff
@@ -6,6 +6,7 @@ import React, {
   useCallback,
 } from 'react'
 import Portal, { Props as PortalProps } from './components/Portal'
+import throttle from 'lodash.throttle'
```

Comment:
I don't know this function very well, so I don't know if there is any gain using it over `debounce`, but if there isn't I think it would be better use `debounce` instead because other [apps already use it](https://github.com/vtex-apps/slider/blob/1d568d18d05878c793783f9d26f499f234f5b452/react/components/Slide.js#L5)

Reply:
@Klynger throttle is similar but different than debounce.

debounce only executes its function once it hasn't been called after X ms.
throttle is fired at most every X ms.

for example, suppose you debounce or throttle a function that is fired on a scroll event:
if debounced, the function is fired only when the user stops scrolling.
if throttled, the function is fired every X ms while the user is scrolling.

in this case, I think throttle is preferred, because it's intended to update the position of the element while the user is scrolling, but not on _every_ frame because that would be too heavy